src/main/java/seedu/manager/model/task/Tag.java

```diff
@@ -23,11 +23,6 @@
      * @throws IllegalValueException if the given tag name string is invalid.
      */
     public Tag(String name) throws IllegalValueException {
-//        assert name != null;
-//        name = name.trim();
-//        if (!isValidTagName(name)) {
-//            throw new IllegalValueException(MESSAGE_TAG_CONSTRAINTS);
-//    }
         super(name, TAG_VALIDATION_REGEX, MESSAGE_TAG_CONSTRAINTS);
```

Comment:
Can in fact remove the isValidTagName method, it's covered in the TaskProperty constructor. Also, then can change the reference to isValidTagName in the header comment to simply isValid. Also, do change the reference to name in the header comment to something like "tag"